vaadinfrontend/frontend/themes/datamanager/components/custom.css

```
@@ -4,6 +4,7 @@
 @import "page-area.css";
 @import "card.css";
 @import "dialog.css";
+@import "projects-collection.css";
```

MAJOR NIT: I'd stay with the singular here since we already established "Experiment-Item-collection"
```suggestion
@import "project-collection.css";
```